noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -3,23 +3,18 @@ contract Test {
     use dep::std::option::Option;
     use dep::aztec::protocol_types::{
         abis::private_circuit_public_inputs::PrivateCircuitPublicInputs,
-        address::{AztecAddress, EthAddress},
         constants::{MAX_READ_REQUESTS_PER_CALL, MAX_NOTES_PER_PAGE}, hash::hash_args
     };
-    // docs:start:unencrypted_import
```

Comment:
I couldn't find a corresponding `#include_code imports /noir-projects/noir-contracts/contracts/test_contract/src/main.nr` btw. Let me know if I missed it.